2-mandatory/3-financial-times.js

```diff
@@ -23,15 +31,26 @@ function titleWithFewestWords(allArticleTitles) {
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+  articlesWithNumbers = [];
+  let digits = /\d+/;
+  for (let i = 0; i < allArticleTitles.length; i++) {
+    if (allArticleTitles[i].match(digits))
+      articlesWithNumbers.push(allArticleTitles[i]);
+  }
+  return articlesWithNumbers;
 }
 
 /*
     The Financial Times wants to understand what the average number of characters in an article title is.
     Implement the function below to return this number - rounded to the nearest integer.
 */
 function averageNumberOfCharacters(allArticleTitles) {
-    // TODO
+  let sum = 0;
```

Comment:
Naming is hard. What is this the sum of? Can you think of a better variable name?